src/base1/patternfly-overrides.less

```diff
@@ -251,6 +251,16 @@ select.form-control {
 
     }
 
+    &.selected > a {
```

Comment:
(Hope I'm commenting in the right place)
I think this also need something for the active state, because if I set active in the web inspector, it will override the `color: var(--pf-global--Color--100);` statement. It's falling back to some statement from dropdowns.less. Doesn't look quite right to me visually.
![Screenshot from 2019-07-08 20-46-38](https://user-images.githubusercontent.com/209895/60834802-cc4eff00-a1c1-11e9-96f2-2fac0278e389.png)

Reply:
Good catch.

The CSS I wrote should work. However, PF3 overrides it with this (compiled) code:

```css
.dropdown-menu>li>a:active{
    background-color:#06c;
    border-color:#0088ce;
    color:#fff!important;
    filter:progid:DXImageTransform.Microsoft.gradient(enabled=false)
}
```

Note the `!important`.

I'm adding an override that also uses `!important` (sigh), because it's needed here.

Reply:
Fixed in fb6cc3b; then squashed it with the other PF commit, rebased on master, and force-pushed up (with the fix).